frontend/src/components/Navbar.jsx

```diff
@@ -3,49 +3,25 @@ import { Link } from "react-router-dom";
 
 export default function Navbar({ onLogout }) {
   return (
-    <nav className="navbar navbar-expand-lg navbar-dark bg-dark">
-      <Link className="navbar-brand" to="/">
-        SDA starter
-      </Link>
-      <button
-        className="navbar-toggler"
-        type="button"
-        data-toggle="collapse"
-        data-target="#navbarColor01"
-        aria-controls="navbarColor01"
-        aria-expanded="false"
-        aria-label="Toggle navigation"
-      >
-        <span className="navbar-toggler-icon"></span>
-      </button>
+    <nav>
+      <Link to="/">Logo</Link>
 
-      <div className="collapse navbar-collapse" id="navbarColor01">
-        <ul className="navbar-nav mr-auto">
-          <li className="nav-item">
-            <Link to="/" className="nav-link">
-              Home
-            </Link>
+      <div>
+        <ul>
+          <li>
+            <Link to="/posts">Home</Link>
           </li>
 
-          <li className="nav-item">
-            <Link to="/posts" className="nav-link">
-              Posts
-            </Link>
+          <li>
+            <Link to="/profile/:id">Profile</Link>
```

Comment:
"profile/id" ?

Reply:
you mean without ":"

Reply:
yes

Reply:
we could have it as "/profile" for now, but as we implement it should point to the current logged user's profile.